Cargo.toml

```diff
@@ -12,7 +12,7 @@ readme = "README.md"
 [features]
 default = ["rayon", "zstd", "libc"]
 bench = ["clap", "num_cpus", "env_logger", "rayon", "zstd"]
-race = ["docopt", "chan-signal", "rand", "rayon", "env_logger"]
+race = ["docopt", "chan-signal", "rand", "rayon"]
```

Comment:
Just a quick question about the `features` definition here.

Should all the dependencies (`extern crate`s) for the binary be listed here?

The [documentation](http://doc.crates.io/manifest.html#the-features-section) is a little bit vague about that.

Why should I list optional dependencies here? For me only "hard" dependencies make sense 🤔